source/include/core_json.h

```diff
@@ -155,15 +156,17 @@ JSONStatus_t JSON_Validate( const char * buf,
  *         value[ valueLength ] = save;
  *     }
  * @endcode
+ *
+ * @note The maximum index value is ~2 billion ( 2^31 - 9 ).
```

Comment:
What is the reason for this max value? May be, an explanatory comment could help

Reply:
The algorithm is the same as you'd find in most any base 10 conversion.  Given our time constraints, I don't want to play the pundit.